Mapbox ironyard

mvaden opened the pull request:
1. 974fc58 - Mapbox API.
2. 58ab10e + eee2dcc - New section/content for location.
4. 1a77ec0 - Toggle between different map styles.
5. 9d5ecb7 - Added IronYard logo to assets directory.
6. b37c98c - Tried and failed to add the IronYard logo to these coordinates. 
7. 5696b91 - Marks the location as listed above.

mvaden committed:
Mapbox API.

mvaden committed:
New Location section now shows Mapbox API and more content.

mvaden committed:
Sectioned off Location area.

mvaden committed:
Radio Buttons.

mvaden committed:
Logo for Mapbox.

mvaden committed:
How to get IronYard logo object to render through the API.

mvaden committed:
Centered the map on IronYard.

impreze commented on index.html:
```diff
@@ -1,228 +1,326 @@
+    <link rel="stylesheet" href="css/main.css">
+
+    <!-- JS - Important to load this in head -->
+    <script src="https://api.mapbox.com/mapbox-gl-js/v0.38.0/mapbox-gl.js"></script>
```
@mvaden Maybe move this down to the JS area [down by line 313](https://github.com/thursdaynetwork/UNhackthevote2017/blob/mapbox_ironyard/index.html#L313)?

mvaden replied:
@impreze I wanted to somehow add the logo as an image to the GeoJson call that object on the map. I tried using this [example](https://www.mapbox.com/mapbox-gl-js/example/add-image/) to get it to work in commit 5696b91. And alternative would be to simply add the logo somewhere within the `location` element. 

When placing that script down on line 313, the browser doesn't render the map at all (see image below). Whereas placing this script in the head does render the map.

 ![mbapi](https://user-images.githubusercontent.com/17731291/27565946-8a30f322-5aaf-11e7-951e-0eedff2a4951.jpeg)

impreze replied:
understood

impreze reviewed (changes requested):
@mvaden what was the issue with the Iron Yard logo? Would you prefer to have it shown next to the location info as opposed to embedding inside the map?

mvaden commented on index.html:
```diff
@@ -1,228 +1,326 @@
+    <link rel="stylesheet" href="css/main.css">
+
+    <!-- JS - Important to load this in head -->
+    <script src="https://api.mapbox.com/mapbox-gl-js/v0.38.0/mapbox-gl.js"></script>
```
@impreze When placing that script down on line 313, the browser doesn't render the map at all (see image below). Whereas placing this script in the head does render the map.

 ![mbapi](https://user-images.githubusercontent.com/17731291/27565946-8a30f322-5aaf-11e7-951e-0eedff2a4951.jpeg)

mvaden committed:
Neatened up the HTML somewhat - maintainablity.

mvaden committed:
SEO.

mvaden committed:
Merge branch 'master' into mapbox_ironyard

impreze committed:
index page was displayed twice for some reason.

Always good to fetch and rebase code to avoid conflicts.

impreze committed:
adjust issues with countdown not rendering

impreze committed:
moved mapbox script to main.js

impreze committed:
reapply html cleanup

impreze committed:
reapply SEO

the reordering of the js files at the bottom was causing some bugs (jquery.js needs to be called first before other jquery libraries)

impreze commented:
@mvaden opposed to this look?
![location](https://user-images.githubusercontent.com/13107842/27570760-b0b2fd6e-5ace-11e7-8ef9-0baed3d88632.png)

mvaden commented:
@impreze Yes - that exactly. This looks great.

mvaden committed:
Updated location address for Iron Yard.

mvaden commented:
@impreze Added the floor (2nd floor) to the location area for 320c8f0. 
The rest looks good from here. 

GTM?

impreze committed:
change location layout and add ironyard logo

impreze committed:
Merge branch 'mapbox_ironyard' of https://github.com/thursdaynetwork/HackTheManderDC2017 into mapbox_ironyard
